medical/models/medical_patient.py

```diff
@@ -107,3 +108,11 @@ def _get_default_image_path(self, vals):
         return get_module_resource(
             'medical', 'static/src/img', 'patient-avatar.png',
         )
+
+    def _search_age(self, operator, value):
+        current_date = date.today()
+        first_possible_birthdate = (current_date -
+                                    relativedelta(years=int(value)+1))
+        possible_dates = [first_possible_birthdate + relativedelta(days=x)
```

Comment:
This is pretty inefficient. Better would be to simply return the operator and the determined date instead of all of this